mods/d2k/rules/arrakis.yaml

```diff
@@ -9,8 +9,7 @@ spicebloom.spawnpoint:
 		GrowthTerrainTypes: SpiceSand
 		SpawnActor: spicebloom
 		GrowthSequences: grow0
-		GrowthDelay: 250, 750
-		RespawnDelay: 1, 2
+		Lifetime: 1750, 3250
```

Comment:
Other than some consideration for 3rd-party mods (wrt changing defaults), no. But this trait won't be used much anyway, I think, so I guess I can make this the new default.